src/components/ui/piechart.tsx

```diff
@@ -0,0 +1,81 @@
+const Progress = ({ progressPercent }: { progressPercent: number }) => {
+  // size of the circle
+  const size = 128;
+  // radius of the circle
+  const radius = 40;
+  // circumference of the circle
+  const circumference = 2 * Math.PI * radius;
+  // strokeDashoffset
+  const strokeDashoffset =
+    circumference - (progressPercent / 100) * circumference;
+
+  return (
+    <>
+      <style>
+        {`@keyframes circleStroke {
+          from {
+            stroke-dashoffset: ${circumference};
+          }
+          to {
+            stroke-dashoffset: ${strokeDashoffset};
+          }
+        }
+        @keyframes rotationObject {
+          from {
+            transform: rotate(0deg);
+          }
+          to {
+            transform: rotate(${progressPercent * 3.6}deg)
+          }
+        }
+        `}
+      </style>
+      <div className="relative">
+        <svg
+          viewBox={`0 0 ${size} ${size}`}
+          style={{ transform: 'rotate(-90deg)' }}
+        >
+          <circle
+            r={radius}
+            cx={size / 2}
+            cy={size / 2}
+            stroke="#CBD5E1"
+            strokeWidth="15"
+            fill="transparent"
+            strokeDasharray={circumference}
+          />
+          <circle
+            r={radius}
+            cx={size / 2}
+            cy={size / 2}
+            stroke="#06B6D4"
+            strokeWidth="15"
+            fill="transparent"
```

Comment:
Could you modify this code with TailwindCSS class and our color class?
`slate-300`

Reply:
same as the one above